tests/nonbonded/test_consistency.py

```diff
@@ -48,6 +51,14 @@ def test_nonbonded_consistency(
     ligand_idxs = rng.choice(num_atoms, size=(num_atoms_ligand,), replace=False).astype(np.int32)
     host_idxs = np.setdiff1d(np.arange(num_atoms), ligand_idxs).astype(np.int32)
 
+    # Partition host_idxs into num_ixn_groups
+    if num_ixn_groups:
+        group_idxs = rng.choice(num_ixn_groups, size=(len(host_idxs),), replace=True).astype(np.int32)
+        col_atom_idxss = [host_idxs[group_idxs == i] for i in range(num_ixn_groups)]
```

Comment:
for my own sanity, maybe rename to `all_col_atom_idxs` to indicate that its a list of `col_atom_idxs`